implementation/ePO_core/owl_ontology/README.md

```diff
@@ -0,0 +1,8 @@
+# owl ontology
+
+**Description**:
+
+
+The *owl_ontology* folder contains the ePO_core ontology files, as well as the restriction files. The files are available in both [RDF](https://www.w3.org/RDF/) and [Turtle](https://www.w3.org/TR/turtle/) formats. 
```

Comment:
'The files are available in both [XML/RDF](https://www.w3.org/TR/rdf-syntax-grammar) and [Turtle/RDF](https://www.w3.org/TR/turtle)'

RDF is the model we are using to represent things, and we use XML or turtle to represent those things. 
I could come also with an RDF graph drawed by hand, the Idea  is to be able to transform this into turtle and then to JsonLD, without loosing information.

This is not to be confused by t

https://www.w3.org/RDF/

Reply:
I see, thank you. I implemented the changes. I ll close this request and make a new one.